Update cdi_segments.md (#10159)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/segments/segment_extension/cdi_segments.md

As part of your CDI setup, you can select from different connections to use in CDI Segment Extensions. Each connection has a specific set of data tables. Your development team can configure your connections and data tables during CDI setup.

> To view the available data tables, including their schema and any available descriptions, select **Reference**. When you're ready, select a connection.

Reviewer: ```suggestion
To view the available your data tables and schema, select **Reference**. When you're ready, select a connection.
```

Reviewer: @colleenkerr , just want to verify that by "their" you were referring to the reader/customer?
